systers_portal/meetup/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from users import SystersUser
+from cities_light.models import City
```

Comment:
Please read PEP8 section about sorting imports in a project (and all PEP8 in general).
First come standard library Python imports. Third party imports come next, separated by a line. The last imports are the project-scope imports. I will let you reformat the imports section by yourself :)